store/modules/learning-resources.ts

```diff
@@ -10,8 +10,8 @@ const TIME_SCALES = Object.freeze({
 
```

Comment:
This is a comment for the line: `any: 'any'`

`any` is a common word used on TS. I think we should replace it with `all` (just for the variable name, not for the label). What do you think?